python/cudf/cudf/window/rolling.py

```diff
@@ -221,7 +221,15 @@ def apply(self, func, *args, **kwargs):
 
           with the exception that math functions in `cmath` are not supported
           since `libcudf` does not have complex number support and output of
-          `cmath` functions are most likely complex numbers
+          `cmath` functions are most likely complex numbers. These five functions
+          in `math` are not supported either, since numba generates multiple PTX
+          functions for them ...
+
+            math.sin()
+            math.cos()
+            math.tan()
+            math.gamma()
+            math.lgamma()
```

Comment:
Don't feel the need to change anything here - but just FYI: typically implementation details such as the above are placed in a `Notes` section of the docstring. See the numpydoc documentation on docstring sections:

https://numpydoc.readthedocs.io/en/latest/format.html#sections